<!doctype html>
<html lang="en-US">
<head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Laboratory of Bioinformatics and Computational Genomics</title>
    <meta name='robots' content='max-image-preview:large' />
    <!-- Other meta tags, styles, and links go here -->

    <!-- You can keep the existing JavaScript and styles if needed -->
</head>

<body class="home page-template-default page page-id-10 wp-custom-logo wp-embed-responsive is-light-theme no-js singular has-main-navigation no-widgets">
    <div id="page" class="site">
        <a class="skip-link screen-reader-text" href="#content">Skip to content</a>
<h2 class="wp-block-heading">Research highlights</h2>



<div class="wp-block-media-text alignwide is-stacked-on-mobile is-style-twentytwentyone-border"><figure class="wp-block-media-text__media"><img decoding="async" width="1024" height="850" src="./Laboratory of Bioinformatics and Computational Genomics_files/drmn_2-1024x850.png" alt="" class="wp-image-49 size-full" srcset="https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/drmn_2-1024x850.png 1024w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/drmn_2-300x249.png 300w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/drmn_2-768x638.png 768w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/drmn_2.png 1383w" sizes="(max-width: 1024px) 100vw, 1024px"></figure><div class="wp-block-media-text__content">
<div class="wp-block-group is-layout-constrained wp-block-group-is-layout-constrained"><div class="wp-block-group__inner-container">
<h2 class="wp-block-heading">Dynamic Regulatory Module networks</h2>



<p>By combining multiple types of regulatory features across multiple cell lines, Dynamic Regulatory Module Networks (DRMNs) can reconstruct context specific regulatory programs to explain gene expression dynamics across these cell lines.</p>



<p><a href="https://doi.org/10.1101/gr.276542.121">https://doi.org/10.1101/gr.276542.121</a></p>
</div></div>
</div></div>



<div class="wp-block-media-text alignwide is-stacked-on-mobile is-style-twentytwentyone-border"><figure class="wp-block-media-text__media"><img decoding="async" width="1024" height="903" src="./Laboratory of Bioinformatics and Computational Genomics_files/hic-1024x903.png" alt="" class="wp-image-50 size-full" srcset="https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/hic-1024x903.png 1024w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/hic-300x265.png 300w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/hic-768x677.png 768w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/hic-1536x1354.png 1536w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/hic-1568x1383.png 1568w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/hic.png 1708w" sizes="(max-width: 1024px) 100vw, 1024px"></figure><div class="wp-block-media-text__content">
<div class="wp-block-group is-layout-constrained wp-block-group-is-layout-constrained"><div class="wp-block-group__inner-container">
<h2 class="wp-block-heading">Arboretum-Hi-C</h2>



<p>Arboretum-Hi-C uses multi-task graph clustering to identify clusters of interacting regions across different samples.</p>



<p><a href="https://doi.org/10.1186/s13059-016-0962-8">https://doi.org/10.1186/s13059-016-0962-8</a></p>
</div></div>
</div></div>



<div class="wp-block-media-text alignwide is-stacked-on-mobile is-style-twentytwentyone-border"><figure class="wp-block-media-text__media"><img decoding="async" width="1024" height="826" src="./Laboratory of Bioinformatics and Computational Genomics_files/merlinp-1024x826.png" alt="" class="wp-image-52 size-full" srcset="https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/merlinp-1024x826.png 1024w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/merlinp-300x242.png 300w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/merlinp-768x619.png 768w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/merlinp-1536x1239.png 1536w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/merlinp-1568x1265.png 1568w, https://lbcg.ut.ac.ir/wp-content/uploads/2023/04/merlinp.png 1840w" sizes="(max-width: 1024px) 100vw, 1024px"></figure><div class="wp-block-media-text__content">
<div class="wp-block-group is-layout-constrained wp-block-group-is-layout-constrained"><div class="wp-block-group__inner-container">
<h2 class="wp-block-heading">MERLIN-P</h2>



<p>MERLIN-P is an extension of MERLIN (Roy et al. 2013) where a probabilistic formulation is used to incorporate the prior belief about regulatory interactions (from different sources e.g. a motif or ChIP-based network) into the process of network reconstruction.</p>



<p><a href="https://doi.org/10.1093/nar/gkw963">https://doi.org/10.1093/nar/gkw963</a></p>
</div></div>
</div></div>

    </div><!-- #page -->
</body>
</html>
